packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -642,11 +502,94 @@ export class Relayer {
+    transactions = transactions.filter((transaction) => {
+      if (!transaction) return false;
+      if (!transaction.transaction) return false;
+      return true;
+    });
+
+    if (transactions.length == 0) return;
+    if (transactions.length == 1) {
+      this.logger.debug({ at: "AcrossRelayerTxProcessor", message: "Sending transaction" });
+      await this._sendTransaction(transactions[0].transaction, transactions[0].message, transactions[0].mrkdwn);
+    }
+    if (transactions.length > 1) {
+      // The `to` field in the transaction must be the same for all transactions or the batch processing will not work.
+      // This should be a MultiCaller enabled contract.
+      const targetMultiCallerContract = new this.l1Client.l1Web3.eth.Contract(
+        getAbi("MultiCaller"),
+        transactions[0].transaction._parent._address
+      );
+
+      for (const transaction of transactions) {
+        if (targetMultiCallerContract.options.address != transaction.transaction._parent._address)
+          throw new Error("Batch transaction processing error! Can't specify multiple `to` fields within batch");
+      }
+
+      // Iterate over all transactions and build up a set of multicall blocks and a block of markdown to send to slack
+      // to make the set of transactions readable.
+      const multiCallTransaction = transactions.map((transaction) => transaction.transaction.encodeABI());
+
+      let mrkdwnBlock = "*Transactions sent in batch:*\n";
+      transactions.forEach((transaction) => {
+        mrkdwnBlock += `  • ${transaction.message}:\n`;
+        mrkdwnBlock += `    - ${transaction.mrkdwn}\n`;
+      });
+
+      // Send the batch transaction to the L1 bridge pool contract. Catch if the transaction succeeds.
+      const batchTxSuccess = await this._sendTransaction(
+        (targetMultiCallerContract.methods.multicall(multiCallTransaction) as unknown) as TransactionType,
+        "Multicall Transaction batch sent!🧙",
+        mrkdwnBlock
+      );
+
+      // In the event the batch transaction was unsuccessful, iterate over all transactions and send them individually.
+      if (!batchTxSuccess) {
+        for (const transaction of transactions) {
+          this.logger.info({ at: "AcrossRelayerTxProcessor", message: "Sending batched transactions individually😷" });
+          await this._sendTransaction(transaction.transaction, transaction.message, transaction.mrkdwn);
+        }
+      }
+    }
+  }
+
+  private async _sendTransaction(transaction: TransactionType, message: string, mrkdwn: string): Promise<boolean> {
+    try {
+      await this.gasEstimator.update();
+      const { receipt } = await runTransaction({
+        web3: this.l1Client.l1Web3,
+        transaction,
+        transactionConfig: { ...this.gasEstimator.getCurrentFastPrice(), from: this.account },
+        availableAccounts: 1,
+      });
+      if (receipt) {
+        this.logger.info({
+          at: "AcrossRelayerTxProcessor",
+          message,
```

Comment:
Is the event receipt still shown?

Reply:
nope! It's up to the `mrkdwn` to contain enough logging information. to inform the user of what happened. in the tx.